.goreleaser.yaml

```
@@ -0,0 +1,63 @@
+version: 1
+
+project_name: lava
+
+gomod:
+  proxy: true
+
+builds:
+  - id: lava
+    env:
+      - CGO_ENABLED=0
+    goos:
+      - linux
+      - windows
+      - darwin
+    main: ./cmd/lava
+    goarch:
+      - amd64
+      - arm
+      - arm64
+    goarm:
+      - "7"
+    ignore:
+      - goos: darwin
+        goarch: arm
+      - goos: windows
+        goarch: arm
+      - goos: windows
+        goarch: arm64
+    mod_timestamp: '{{ .CommitTimestamp }}'
+    ldflags:
+      - -s -w -X main.version={{.Version}} -X main.commit={{.Commit}} -X main.date={{.Date}}
+        -X main.builtBy=goreleaser
```

We are reading the command version in runtime, so we don't need most of these flags.

```suggestion
    ldflags:
      - -s -w
```